src/sidebar.ts

```diff
@@ -500,68 +440,93 @@ const identitiesSidebar = [
       "security-model",
       "identities/native-browser",
       "concepts/redirects",
+      {
+        type: "category",
+        label: "User flows",
+        link: {
+          type: "doc",
+          id: "kratos/self-service",
+        },
+        items: [
+          "kratos/concepts/browser-redirect-flow-completion",
+          "kratos/self-service/flows/user-registration",
+          "kratos/self-service/flows/user-login",
+          "kratos/self-service/flows/user-logout",
+          "kratos/self-service/flows/user-settings",
+          "kratos/self-service/flows/verify-email-account-activation",
+          "kratos/self-service/flows/account-recovery-password-reset",
+          "kratos/self-service/flows/user-facing-errors",
+        ],
+      },
     ],
   },
   {
     type: "category",
-    label: "Self-service user flows",
-    collapsed: false,
-    collapsible: false,
-    link: {
-      type: "doc",
-      id: "kratos/self-service",
-    },
-    items: [
-      "kratos/concepts/browser-redirect-flow-completion",
-      "kratos/self-service/flows/user-registration",
-      "kratos/self-service/flows/user-login",
-      "kratos/self-service/flows/user-logout",
-      "kratos/self-service/flows/user-settings",
-      "kratos/self-service/flows/verify-email-account-activation",
-      "kratos/self-service/flows/account-recovery-password-reset",
-      "kratos/self-service/flows/user-facing-errors",
-    ],
-  },
-  {
-    type: "category",
-    label: "Authentication",
+    label: "Guides",
     collapsed: false,
     collapsible: false,
-    link: {
-      type: "doc",
-      id: "kratos/concepts/credentials",
-    },
     items: [
-      "concepts/password-policy",
-      "kratos/passwordless/passwordless",
-      "kratos/mfa/overview",
+      "getting-started/local-development",
+      {
+        type: "category",
+        label: "Authenctication",
+        items: [
+          "getting-started/local-development",
+          "kratos/concepts/credentials/username-email-password",
+          "kratos/passwordless/passkeys",
+          "kratos/passwordless/one-time-code",
+          "kratos/organizations/organizations",
+          "kratos/emails-sms/custom-email-templates",
+        ],
+      },
+      {
+        type: "category",
+        label: "Multi-factor authentication",
+        link: {
+          type: "doc",
+          id: "kratos/mfa/overview",
+        },
+        items: [
+          "kratos/mfa/lookup-secrets",
+          "kratos/mfa/totp",
+          "kratos/mfa/webauthn-fido-yubikey",
+          "kratos/mfa/mfa-via-sms",
+          "kratos/mfa/step-up-authentication",
+        ],
+      },
+      {
+        type: "category",
+        label: "Credentials",
```

Comment:
wouldn't the other auth methods like `password`, `passkeys` etc. also fall under credentials? 
not sure what I would except in this category

Reply:
Thant's a good catch. Thanks